### PR TITLE
error handling implemented

### LINKED ISSUE #25: Unhandled deposit error for ETH
When the ETH deposit contract execution has been failed with insufficient fund, it doesn't shown on userland.

Screen Shot 2019-04-09 at 16 26 40

### PR DESCRIPTION
**Pull request by shogochiai**
This pull-req closes #25

### CODE REVIEW

File: src/redux/modules/chamberWallet/deposit.ts [hunk 1]
```diff
@@ -92,19 +92,28 @@ export const deposit = (
...
-  const result =
+
+  try {
+    result =
     token.id === 0
       ? await ref.deposit(amount.toString())
       : await ref.depositERC20(token.address, amount)
+  } catch(e){
+    isError = true
+    errorMsg = e.message
```
Comment (shogochiai) [inline src/redux/modules/chamberWallet/deposit.ts]: Not good for i18n

File: src/redux/modules/chamberWallet/deposit.ts [hunk 2]
```diff
@@ -92,19 +92,28 @@ export const deposit = (
...
+  }
 
-  if (result.isOk()) {
+  if (result && result.isOk() && isError) {
```
Comment (daiki44) [inline src/redux/modules/chamberWallet/deposit.ts]: Will `isError` be false?

Comment (shogochiai) [reply src/redux/modules/chamberWallet/deposit.ts]: Ouch, boolean reversed.

```
if (result && result.isOk() && !isError) {
   dispatch(depositSuccess())
  } else {
    errorMsg = (errorMsg) ? errorMsg : result.error()
    dispatch(depositFail(errorMsg))
  }	  
}
```

**Comment by tkmct** [comment]
一旦close

**Pull request closed without merging**